.continuous-integration/travis/setup_dependencies_common.sh

```diff
@@ -53,9 +53,9 @@ fi
 # build_sphinx needs sphinx as well as matplotlib and wcsaxes (for plot_directive). 
 if [[ $SETUP_CMD == build_sphinx* ]]
 then
-  $CONDA_INSTALL Sphinx Pygments matplotlib
-  pip install wcsaxes
+  $CONDA_INSTALL matplotlib
   pip install sphinx_rtd_theme
```

Comment:
travis-ci passed with what's here now: https://travis-ci.org/astropy/astroplan/builds/82747045
But OK, I'll try conda installing Sphinx and sphinx_rtd_theme instead ... if it works it could be a bit faster / more robust.